test/e2e/v3/generate_test.go

```diff
@@ -229,3 +229,101 @@ Count int `+"`"+`json:"count,omitempty"`+"`"+`
 		_ = pluginutil.RunCmd("Update dependencies", "go", "mod", "tidy")
 	}
 }
+
+// GenerateV4 implements a go/v4(-alpha) plugin project defined by a TestContext.
+func GenerateV4(kbc *utils.TestContext, crdAndWebhookVersion string) {
+	var err error
+
+	By("initializing a project")
+	err = kbc.Init(
+		"--plugins", "go/v4",
```

Comment:
Then,. when we properly ensure that the go/v4 ( bundle plugin in the cli.go ) be == alpha we will need to change the key. IT will use alpha as well.